### PR TITLE
Add ip command

### PR DESCRIPTION
**Pull request by Sispheor**
Everything is in the title ;)

### CODE REVIEW

**Comment by srsudar** [comment]
Thanks! I'll work on merging this and the other PR starting at the end of next week.

**Comment by scorphus** [comment]
@Sispheor Thanks a lot for you contribution! I have some line comments, though.

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,6 @@ syntax: glob
 *.egg-info
 dist
 build
+
+# Pycharm
+.idea
```
Comment (scorphus) [inline .gitignore]: These are IDE/editor-specific files, which differ from user to user, as our personal choices are diverse. The right place for these – and many others – is the global ignore file `~/.config/git/ignore`, following the XDG Base Directory Specification. `man gitignore` has detailed information.

File: eg/examples/ip.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+Set a Default Route
+    
+    ip route add default via 10.0.0.254
+
+Delete Routes from the Routing table
+    
+    ip route delete 192.168.1.0/24 dev eth0
+    
+# Basic Usage
```
Comment (scorphus) [inline eg/examples/ip.md]: Examples use to have three lines before a heading.

**Comment by scorphus** [comment]
Also, would you please squash the commits into a single one?

**Comment by Sispheor** [comment]
Should be ok now

**Comment by scorphus** [comment]
Oh. Actually, not quite. Below is how the tree looks now ([click here](https://github.com/scorphus/dotfilesetal/blob/master/.gitconfig#L49) for the Git alias that generated the following):

clip

If you'd like, and allow me, I can record an [asciinema](http://asciinema.org) while I *fix* both branches, amending/squashing your commits and commenting as I do it.

Then, if you grant me write permission to your Sispheor/eg repository, I can fix the pull requests as well. It's like an image/video is worth many words :relaxed:

**Comment by Sispheor** [comment]
I've added you as collaborator on my project.

2015-11-17 0:11 GMT+01:00 Pablo Aguiar <notifications@github.com>:

> Oh. Actually, not quite. Below is how the tree looks now (click here
> <https://github.com/scorphus/dotfilesetal/blob/master/.gitconfig#L49> for
> the Git alias that generated the following):
>
> [image: clip]
> <https://cloud.githubusercontent.com/assets/2112697/11197475/de158dac-8ca4-11e5-801f-d0eb4a66a870.png>
>
> If you'd like, and allow me, I can record an asciinema
> <http://asciinema.org> while I *fix* both branches, amending/squashing
> your commits and commenting as I do it.
>
> Then, if you grant me write permission to your Sispheor/eg repository, I
> can fix the pull requests as well. It's like an image/video is worth many
> words [image: :relaxed:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/srsudar/eg/pull/46#issuecomment-157204359>.
>



-- 
*Nicolas MARCQ*

**Comment by scorphus** [comment]
@Sispheor I've updated the branch `add_ip_command` which is the result of the following recorded terminal session:

[asciicast](https://asciinema.org/a/216bruwcrq5s3x1ctyeewv5n5)

I hope it's helpful :smiley: 

I'll soon update the `add_git_command` branch too, but with a different approach.

**Comment by Sispheor** [comment]
Thank you for the job. And for the Git lesson! ;)

2015-11-17 16:18 GMT+01:00 Pablo Aguiar <notifications@github.com>:

> @Sispheor <https://github.com/Sispheor> I've updated the branch
> add_ip_command which is the result of the following recorded terminal
> session:
>
> [image: asciicast] <https://asciinema.org/a/216bruwcrq5s3x1ctyeewv5n5>
>
> I hope it's helpful [image: :smiley:]
>
> I'll soon update the add_git_command branch too, but with a different
> approach.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/srsudar/eg/pull/46#issuecomment-157400754>.
>



-- 
*Nicolas MARCQ*

**Comment by scorphus** [comment]
> Thank you for the job. And for the Git lesson! ;)

My pleasure! I'm glad it helped even if a bit :blush:

**Pull request closed without merging**